tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -113,6 +113,12 @@ func newSession(credentialPath, credentialAccountID, region string) (*awssession
 	})
 }
 
+// CreateWindowsVMWithPrivate creates a WindowsVM with private subnet of OpenShift cluster. A public
```

Comment:
Doc name mismatch

Reply:
`within a private subnet of the OpenShift cluster`?

Reply:
Thank you, addressed it.